Add course page and material for Intro to ChIP-Seq (#68)
stjude-dnb-binfcore/trainings: courses/ChIP-Seq/README.md

<p align="center">
  <img src="img/DNB-BINF-Core-logo.png" alt="DNB-BINF-Core logo" width="560px" />
</p>

# Introduction to ChIP Sequencing and QC Workshop

The `Introduction to ChIP Sequencing and QC` workshop aims to provide attendees with an overview of the methods and steps in ChIP sequencing, data analysis, and quality control. In particular, the workshop focuses on the ChIP-Seq mapping, QC, and analysis workflow utilized by the Center for Applied Bioinformatics (CAB) at St. Jude Children's Research Hospital.

Please see the [CAB’s QC and Peak Calling pipeline](https://wiki.stjude.org/display/CAB/ChIPseq+QC+and+peak+calling) for more information.

| Audience | Computational Skills | Prerequisites | Duration |
:----------|:----------|:----------|:----------|
| Biologists/Bioinformaticians | Beginner | None | 45 min|

## Access to the workshop recording

> St. Jude employees can access a [recording from the workshop](https://sjcrh-my.sharepoint.com/:v:/g/personal/sfreshou_stjude_org/EfAqM3KcgRlNho8TXNckL80Bl1JPTsb-k3F_t8W6c-Hewg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=ejIZLy).

Reviewer: ```suggestion
St. Jude employees can access the [recording from the workshop](https://sjcrh-my.sharepoint.com/:v:/g/personal/sfreshou_stjude_org/EfAqM3KcgRlNho8TXNckL80Bl1JPTsb-k3F_t8W6c-Hewg?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=ejIZLy).
```

## Below is the main directory structure listing the analyses and data files used in this repository

```
├── lectures

> |   ├── Introduction-to-ChIP-Sequencing-and-QC-2024-12-11.pdf

Reviewer: ```suggestion
|   └── Introduction-to-ChIP-Sequencing-and-QC-2024-12-11.pdf
```

└── README.md
```

## Course Authors

Sharon Freshour, PhD ([@sharonfreshour](https://github.com/sharonfreshour))

## Contact

> Contributions, issues, and feature requests are welcome! Please feel free to check [issues](https://github.com/stjudeDNBBinfCore/Trainings/issues).

Reviewer: ```suggestion
Contributions, issues, and feature requests are welcome! Please feel free to check [issues](https://github.com/stjude-dnb-binfcore/trainings/issues).
```
